packages/wonder-blocks-data/src/util/response-cache.js

```diff
@@ -9,6 +9,8 @@ import type {
     ResponseCache as ResCache,
 } from "./types.js";
 
+const DefaultScope = "default";
```

Comment:
Since the new in-memory cache is scoped and we don't need that, we just use a default value for scope.

Reply:
If we find ourselves using single-scope caches in lots of places, we may want to provide a facade where specifying the scope isn't necessary.  No action req'd.

Reply:
Yes. I did have one, but I felt it wasn't being very helpful and just bloated the WB Data package unnecessarily.